extensions/wikia/YouTube/YouTube.php

```diff
@@ -225,9 +225,9 @@ function wfYouTube( Parser $parser ): bool {
 function embedYouTube_url2ytid( $url ) {
 	$id = $url;
 
-	if ( preg_match( '/^http:\/\/www\.youtube\.com\/watch\?v=(.+)$/', $url, $preg ) ) {
+	if ( preg_match( '/^https?:\/\/www\.youtube\.com\/watch\?v=(.+)$/', $url, $preg ) ) {
 		$id = $preg[1];
-	} elseif ( preg_match( '/^http:\/\/www\.youtube\.com\/v\/([^&]+)(&autoplay=[0-1])?$/', $url, $preg ) ) {
+	} elseif ( preg_match( '/^https?:\/\/www\.youtube\.com\/v\/([^&]+)(&autoplay=[0-1])?$/', $url, $preg ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This branch's code block is the same as the block for the branch on line 228. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)

[project-id]: wikia:app